features/bootstrap/SwaggerContext.php

```diff
@@ -0,0 +1,260 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Behat\Behat\Context\Context;
+use Behat\Behat\Context\Environment\InitializedContextEnvironment;
+use Behat\Behat\Hook\Scope\BeforeScenarioScope;
+use Symfony\Component\PropertyAccess\PropertyAccess;
+
+class SwaggerContext implements Context
```

Comment:
@dunglas is it necessary? It's a context so it shouldn't be part of the classmap if installed with dev dependencies. We're not writing our test cases as final either